src/test/reservationCounter.test.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import InvolvementService from '../modules/InvolvementService.js';
+
+describe('Testing reservation counter', () => {
```

Comment:
- [ ] Please create a new test for the implemented counter. I'm sure you will do your best.